packages.dhall

```diff
@@ -99,35 +99,6 @@ in  upstream
 -------------------------------
 -}
 let upstream =
-      https://github.com/purescript/package-sets/releases/download/psc-0.14.4-20210905/packages.dhall sha256:140f3630801f2b02d5f3a405d4872e0af317e4ef187016a6b00f97d59d6275c6
+      https://github.com/purescript/package-sets/releases/download/psc-0.14.5-20211111/packages.dhall sha256:7ed6350fe897a93926d16298e37d2324aabbe5eca99810204719dc3632fb555f
 
 in  upstream
-  with canvas-action =
```

Comment:
We no longer need this override as we can use the version of `canvas-action` from the package set.

The only difference between the version in the package set and the version that was previously referenced is a README change: https://github.com/artemisSystem/purescript-canvas-action/compare/v7.0.0...43de19ee369d1ff9fe7eff1e583b828809fd9e36